students/17th/team-one/heeyoungjeong/user/views.py

```diff
@@ -0,0 +1,41 @@
+import json
+from django.http import JsonResponse, HttpResponse
+from django.views import View
+from json.decoder import  JSONDecodeError
+from user.models import User
+
+class SignUp(View):
+    def post(self, request):
+
+        try:
+            data = json.loads(request.body)
+
+            check_lst = ['name', 'user_name', 'email', 'password']
+            for key in check_lst:
+                if key not in data.keys():
+                    return JsonResponse({'message':'KEY_ERROR'}, status=400)
+
+            if not '@' in data['email'] or not '.' in data['email']:
+                return JsonResponse({'message':'The email is not valid'}, status=400)
+
+            if not 8 <= len(data['password']):
```

Comment:
```suggestion
            if not MINIMUM_PASSWORD_LENGTH <= len(data['password']):
```
패스워드의 최소 허용길이라면 이값도 하드코딩할게 아니라 상수화 해주세요~!

Reply:
수정하였습니다.
하드코딩 피하도록 하고 상수화 사용하도록 하겠습니다.